samples/ml-ops-on-aws-img-recognition/source/containers/ml-ops-byo-resnet18/Dockerfile

```
@@ -0,0 +1,21 @@
+FROM python
```

Best practice to pick a version, in case future versions have breaking changes.

```suggestion
FROM python:3.8
```

---

Good call. I added the python version.